jenkins/Dockerfile-blossom.integration.centos

```diff
@@ -17,13 +17,13 @@
 ###
 #
 # Arguments:
-#    CUDA_VER=10.1, 10.2 or 11.0
+#    CUDA_VER=11.0, 11.1 or 11.2.2
 #    CENTOS_VER=7 or 8
-#    CUDF_VER=0.18 or 0.19
+#    CUDF_VER=0.19 or 0.20
```

Comment:
Can it use 0.19 cuDF?

Reply:
this is just the arg for the image itself, we use it for both pre_release and dev pipelines.